pkg/sql/plan/base_binder.go

```diff
@@ -764,7 +764,7 @@ func bindFuncExprImplByPlanExpr(name string, args []*Expr) (*plan.Expr, error) {
 		if len(args) != 2 {
 			return nil, moerr.NewInvalidArg("date_add/date_sub function need two args", len(args))
 		}
-		args, err = resetDateFunctionArgs(args[0], args[1])
+		args, err = resetDateFunctionArgs2(args[0], args[1])
```

Comment:
Really bad names, what is this 2 for?   Seems the no-2 function takes two arguments anyway ...

Reply:
The function was written by others， i just use the function